test/suites/basic.sh

```diff
@@ -105,6 +105,21 @@ test_interactive() {
     validate_system_microceph "${m}" 1 disk2
     validate_system_microovn "${m}"
   done
+
+  # Reset the systems and install microovn and microceph (with Ceph encryption).
+  reset_systems 3 3 1
+
+  echo "Creating a MicroCloud with ZFS and Ceph storage, and OVN network"
+  export CEPH_ENCRYPT="yes"
+  microcloud_interactive | lxc exec micro01 -- sh -c "microcloud init > out"
+
+  lxc exec micro01 -- tail -1 out | grep "MicroCloud is ready" -q
+  for m in micro01 micro02 micro03 ; do
+    validate_system_lxd "${m}" 3 disk1 3 1 "${OVN_FILTER}" "${IPV4_SUBNET}" "${IPV4_START}"-"${IPV4_END}" "${IPV6_SUBNET}" "${DNS_ADDRESSES}"
+    validate_system_microceph "${m}" 1 disk2
+    validate_ceph_encrypt "${m}" disk2
```

Comment:
Maybe move `validate_ceph_encrypt` into `validate_system_microceph` with a new argument before or after the `CephFS` argument which indicates whether we expect encryption? That way it will be checked for every test.

Reply:
fixed